test/IRGen/alloc.sil

```diff
@@ -19,6 +21,8 @@ struct Huge {
   var unalign: Builtin.Int8
 }
 
-// CHECK: define linkonce_odr hidden i8* @__swift_memcpy4097_8(i8*, i8*, %swift.type*)
+// CHECK-COMDAT: define linkonce_odr hidden i8* @__swift_memcpy4097_8(i8*, i8*, %swift.type*)
+// CHECK-COMDAT-SAME: comdat
+// CHECK-NOCOMDAT-NOT:comdat
```

Comment:
This will only check that "comdat" isn't written *here,* not that it isn't written at all. Did you mean to leave the first line as CHECK instead of CHECK-COMDAT?

Reply:
Oops, yeah, the first line should be `CHECK`.